lib/tasks/reports.rake

```diff
@@ -31,14 +31,13 @@ namespace :reports do
   task :expire => :environment do
     conditions = {}
     conditions[:timerange] = ENV['days'].to_i.days if ENV['days']
-    conditions[:status] = ENV['status'].to_i if ENV['status']
-    batch_size = 1000
+    batch_size = 100_000
     batch_size = ENV['batch_size'].to_i if ENV['batch_size']
     sleep_time = 0.2
     sleep_time = ENV['sleep_time'].to_f if ENV['sleep_time']
 
     User.as_anonymous_admin do
-      report_type.expire(conditions, batch_size, sleep_time)
+      ConfigReport.expire(conditions, batch_size, sleep_time)
```

Comment:
what if i only want to expire a different report type, such as ARF reports?

Reply:
My proposal is to delete all types, user is no longer allowed to expire reports per type. This is actually incorrect, changing this to `Report.expire`.